llvm/lib/Target/AIE/AIEInterBlockScheduling.cpp

```diff
@@ -485,9 +485,11 @@ int InterBlockScheduling::getCyclesToRespectTiming(
     for (auto &Bundle : R.Bundles) {
       for (MachineInstr *MI : Bundle.getInstrs()) {
         DistancesFromLoopEntry[MI] = DistFromLoopEntry;
-        Edges.addNode(MI);
       }
-      ++DistFromLoopEntry;
```

Comment:
curious: any reason to change this from pre-increment to post-increment?

Reply:
Hi, good eyes. Probably I deleted this increment and retyped it again, ! I will change it to be the same as before.